.github/workflows/hardware-long.yml

```diff
@@ -11,7 +11,12 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  build-fw-artifact:
```

Comment:
Generally looks good- the github annotations here should provide a hint, but you need to remove the trailing whitespace here to pass compliance checks